// [START declaration]
syntax = "proto3";
// Namespace
package greetings;
// [END declaration]

// [START service]
// The greeting service definition.
service Greeter {
    // Sends a greeting
    rpc SayHello (HelloRequest) returns (HelloReply) {}
}
// [END service]

// [START messages]
// The request message containing the user's name.
message HelloRequest {
    string name = 1;
}

// The response message containing the greetings.
message HelloReply {
    string message = 1;
}
// [END messages]